### DISCUSSION TITLE
Add option to preview files in a modal within terminal.

### DISCUSSION THREAD

**Discussion by austinoneil** [d99kris/nchat#307] [Ideas]
It would be nice to be able to preview a file within the terminal using a modal (similar to what is used by the file selector).

API:
* Add a flag for previewing in the modal in ui.conf
* Display the output of the `attachment_open_command` within that modal.

This is useful for viewing images within terminal using `catimg` or viewing audio files using one of the various terminal media players.

**Comment by d99kris** [comment]
Thanks for suggesting a feature. I don't anticipate I'll be able to look into this anytime soon, but I'll leave the request here for others to comment / upvote.

**Comment by Bellavene** [comment]
You can do it already, I use ranger for that. Just tell it to start with preview already engaged.

**Reply by TBS1996** [reply]
could you show how that's done? :)
